### PR TITLE
Delete environment variables after used

### PR DESCRIPTION
**Pull request by ybiquitous**
> Explain a summary, purpose, or background for this change.

For security, this change aims to delete the environment variables after used.

> Link related issues or pull requests.

None.

> Check the following items.

- [x] Add a new [changelog](https://github.com/sider/runners/blob/master/CHANGELOG.md) entry if this change is notable.

### CODE REVIEW

File: bin/runners [hunk 1]
```diff
@@ -13,12 +13,13 @@ EXIT_STATUS_FOR_SIGTERM = 126
 
 # @see https://docs.bugsnag.com/platforms/ruby/configuration-options
 Bugsnag.configure do |config|
-  config.app_version = ENV["RUNNERS_VERSION"]
-  config.release_stage = ENV["BUGSNAG_RELEASE_STAGE"]
+  config.api_key = ENV.delete("BUGSNAG_API_KEY")
+  config.app_version = ENV.delete("RUNNERS_VERSION")
+  config.release_stage = ENV.delete("BUGSNAG_RELEASE_STAGE")
```
Comment (ybiquitous) [inline bin/runners:18]: [note] https://ruby-doc.org/core-2.7.1/ENV.html#delete-method

**Comment by yykamei** [comment]
Thank you :smile:  I'm reading this PR. Please wait for a minute :pray:

File: bin/runners [hunk 2]
```diff
@@ -33,7 +34,18 @@ trap('SIGUSR2') do
 end
 
 begin
-  Runners::CLI.new(argv: ARGV.dup, stdout: STDOUT, stderr: STDERR).run
+  # @see https://docs.aws.amazon.com/sdk-for-ruby/v3/developer-guide/setup-config.html
+  Aws.config.update({
+    credentials: Aws::Credentials.new(ENV.delete("AWS_ACCESS_KEY_ID"), ENV.delete("AWS_SECRET_ACCESS_KEY")),
+    region: ENV.delete("AWS_REGION"),
+  })
```
Comment (ybiquitous) [inline bin/runners:41]:
```suggestion
  if ENV["AWS_ACCESS_KEY_ID"]
    Aws.config.update({
      credentials: Aws::Credentials.new(ENV.delete("AWS_ACCESS_KEY_ID"), ENV.delete("AWS_SECRET_ACCESS_KEY")),
      region: ENV.delete("AWS_REGION"),
    })
  end
```

Comment (ybiquitous) [reply bin/runners:41]: @yykamei How about this solution? 👇 

```suggestion
  if ENV["AWS_ACCESS_KEY_ID"]　
    Aws.config[:credentials] = Aws::Credentials.new(ENV.delete("AWS_ACCESS_KEY_ID"), ENV.delete("AWS_SECRET_ACCESS_KEY"))
  end
  if ENV["AWS_REGION"]
    Aws.config[:region] = ENV.delete("AWS_REGION")
  end
```

Comment (yykamei) [reply bin/runners:41]: It looks good. Thank you.

**Review by yykamei — APPROVED** [review]
LGTM

**Pull request merged**